realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -158,6 +160,16 @@ private Realm(RealmCache cache) {
     }
 
     /**
+     * Returns the immutable schema for this Realm.
```

Comment:
I would probably rephrase this to.
```
Returns the schema for this Realm. The schema is immutable. 
Any attempt to modify it will result in an {@link UnsupportedOperationException}.
<p>
The schema can only be modified using {@link DynamicRealm.getSchema()} or through an migration.
```